src/main/java/io/github/ibuildthecloud/gdapi/validation/ResourceManagerReferenceValidator.java

```diff
@@ -37,6 +37,24 @@ public Object getByField(String type, String fieldName, Object value) {
         return list.size() == 0 ? null : list.get(0);
     }
 
+    @Override
+    public Object getByFieldExcludeById(String type, String fieldName, Object value, String id) {
```

Comment:
I can just wasn't sure if ```getByField``` was used in other places/ if you would be ok with renaming it to ```getByFieldExcludeById```